extensions/amp-list/amp-list.md

```diff
@@ -41,6 +41,10 @@ You can specify a template in one of two ways:
 - a `template` attribute that references an ID of an existing templating element.
 - a templating element nested directly inside the `amp-list` element.
 
+[tip type="note"]
+When using `<amp-list>` in tandem with some other templating AMP component, such as `<amp-form>`, note that templates may not nest in valid AMP documents. In this case a valid workaround is to provide the template by `id` via the `template` attribute. Learn more about [nested templates in `<amp-mustache>`](https://amp.dev/documentation/components/amp-mustache/#nested-templates).
```

Comment:
nit: "in tandem with another templating AMP component" might sound a better, up to you if you want to change it.  Same comment for both changes.